hoot-core/src/main/cpp/hoot/core/util/SignalCatcher.cpp

```diff
@@ -204,8 +204,9 @@ void SignalCatcher::terminateHandler()
   try
   {
     // try once to re-throw currently active exception
-    if (!tried_throw++)
+    if (!tried_throw)
```

Comment:
Using the pre/postfix `++` operator on a `bool` is not cool.  (see what I did there?)